### DISCUSSION TITLE
0.12.0

### DISCUSSION THREAD

**Discussion by tillsteinbach** [tillsteinbach/VWsFriend#125] [General]
### Added
- Recording of trips for ID vehicles

### Changed
- Interval can be decresed to 180 seconds
- Technical fields are now hidden in the tables
- Removed interpolation from charging graph

<hr /><em>This discussion was created from the release <a href='https://github.com/tillsteinbach/VWsFriend/releases/tag/v0.12.0'>0.12.0</a>.</em>

**Comment by Julien4135** [comment]
Hello, stupid question, but how can I be sure that I am up to date with version 0.12.0 ?
Otherwise I have to copy paste the entire file by modifying .env with the correct login ?

**Reply by tillsteinbach** [reply]
Actually you just have to do a `docker-compose pull` To fetch the latest version. You can also use watchtower for automatic updates.
